index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+
+<body>
+    <header class="header-container">
+        <nav class="nav-item1">My logo</nav>
```

Comment:
- [ ] Kindly use `a` tag for the logo. 
change `<nav class="nav-item1">My logo</nav>` to `<a href="#" class="nav-item1">My logo</nav>`